pkg/cmd/server/api/v1/conversions.go

```diff
@@ -131,6 +131,10 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 			if len(obj.BindNetwork) == 0 {
 				obj.BindNetwork = "tcp4"
 			}
+			if obj.AllowRecursiveQueries == nil {
```

Comment:
So, this only goes in after nodes run dns on their own so they no longer need to connect to master's skydns? Trying to understand why we didn't simply enable recursive lookups long ago like upstream does.